inference-engine/src/multi_device/multi_device_plugin.cpp

```diff
@@ -234,9 +241,10 @@ IExecutableNetworkInternal::Ptr MultiDeviceInferencePlugin::LoadNetworkImpl(cons
     }
     // check if it is -d AUTO or -d AUTO:xPU use case
     if (workMode != fullConfig.end()) {
-        auto targetDevice = SelectDevice(metaDevices, networkPrecision);
-        // std::cout << "!!! DEBUG: select device is " << targetDevice.deviceName << std::endl;
-        metaDevices = { targetDevice };
+        // select the device
+        auto device = SelectDevice(metaDevices, networkPrecision).deviceName;
+        // parse the config for the device
+        metaDevices = ParseMetaDevices(SelectDevice(metaDevices, networkPrecision).deviceName, fullConfig);
```

Comment:
Merging MULTI and AUTO is WIP. Parse fullConfig is aligned with current master.

Finally AUTO will only filter hints instead full config to target device as design.